trip_app/trips/views.py

```diff
@@ -43,3 +53,30 @@ def get_queryset(self):
                 order_by(F('dist1') + F('dist2'))
 
         return queryset
+
+    @action(detail=True, methods=['post'])
+    def reserve(self, request, pk=None):
+        """ Reserve a trip by sending a POST to api/trip/<trip_id>/reserve/.
```

Comment:
Format docstring. Delete unnecessary information from it.

Reply:
Genuine question, what information do you consider unnecessary in this doc string ?

Reply:
There is no need in this : ```
If the man_approve field of the trip model is True the trip request will be created,
            which means that user requested a trip should be approved by the driver.
            If man_approve is False user will be added to the passengers list if there are free seats
``` because all this information can be understood from code below.

Reply:
Removed mentioned comments.